server/handler/HookHandler.js

```diff
@@ -50,7 +51,10 @@ class HookHandler {
         await this.githubService.readZapprFile(owner.login, name, repo.checks[0].token)
         : {}
 
-      const config = merge({}, DEFAULT_CONFIG, zapprUserConfig)
+      const config = mergeWith({}, DEFAULT_CONFIG, zapprUserConfig, (objValue, srcValue) => {
```

Comment:
We can shorten this to

~~~ javascript
const config = mergeWith({}, DEFAULT_CONFIG, zapprUserConfig, (objValue, srcValue) => isArray(srcValue) ? srcValue : undefined)
~~~